shared/src/main/scala/mouse/any.scala

```diff
@@ -0,0 +1,13 @@
+package mouse
+
+
+trait AnySyntax {
+  implicit final def optionSyntaxMouse[A](oa: A): AnyOps[A] = new AnyOps(oa)
+}
+
+final class AnyOps[A](val oa: A) extends AnyVal {
+  def |>[B] (f: A => B) = f(oa)
+  def thrush[B] (f: A => B) = f(oa)
+
+  def ignore = ()
```

Comment:
Good point. I have made this change. Due to conflicts in FunSuite inherited through MouseSuite, this isn't as obvious in the tests, but it works, I believe.